spec/models/trainee_spec.rb

```diff
@@ -23,5 +23,21 @@
         expect { trainee.dttp_id = uuid }.to raise_error(LockedAttributeError)
       end
     end
+
+    context "record set up validation" do
+      it "tests for the presence of a record type of new trainee" do
+        trainee = Trainee.new
+
+        expect(trainee.valid?).to be false
+        expect(trainee.errors.keys).to include(:record_type)
+      end
+
+      it "tests for the 100 character limit on trainee id" do
+        trainee = Trainee.new(record_type: "assessment_only", trainee_id: SecureRandom.alphanumeric(101))
```

Comment:
We can use the same thing here. Or if `subject` gets unwieldy we should use the FactoryBot factory.